source/usage-examples/insertOne.txt

```diff
@@ -0,0 +1,27 @@
+=================
+Create a Document
+=================
+
+.. default-domain:: mongodb
+
+You can create a single document using the `insertOne()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#insertOne>`_
+method. The ``insertOne()`` method inserts a document into the specified collection
+using a document object. If the specified collection does not exist, the ``insertOne()``
```

Comment:
What is a document object? I think you mean `insertOne(<Document Object>)` - I'd omit the "object" portion:

"inserts a document in the specified collection"

Reply:
Fixed